RELEASE.rst

```diff
@@ -1,3 +1,139 @@
+Release 0.1.4
+=============
+
+Library name change
+-------------------
+The library and GitHub repository have been renamed to `giotto-tda`! While the
+new name is meant to better convey the library's focus on Topology-powered
+machine learning and Data Analysis, the commitment to seamless integration with
+`scikit-learn` will remain just as strong and a defining feature of the project.
+Concurrently, the main module has been renamed from `giotto` to `gtda` in this
+version. `giotto-learn` will remain on PyPI as a legacy package (stuck at v0.1.3)
+until we have ensured that users and developers have fully migrated. The new PyPI
+package `giotto-tda` will start at v0.1.4 for project continuity.
+
+TL;DR ::
```

Comment:
Is this standard?

Reply:
Which part of this do you mean?

Reply:
The `TL;DR ::` part

Reply:
If you think it's too colloquial, I'm happy to change to something more sombre: suggestions?

Reply:
How about something like “Short summary”? TL;DR might not be obvious to non-English speakers and / or people over a certain age like me ;-)

Reply:
Done!